ceph.spec.in

```diff
@@ -485,7 +485,7 @@ Recommends:	ceph-mgr-diskprediction-local = %{_epoch_prefix}%{version}-%{release
 Recommends:	ceph-mgr-diskprediction-cloud = %{_epoch_prefix}%{version}-%{release}
 Recommends:	ceph-mgr-rook = %{_epoch_prefix}%{version}-%{release}
 Recommends:	ceph-mgr-k8sevents = %{_epoch_prefix}%{version}-%{release}
-Recommends:	ceph-mgr-ssh = %{_epoch_prefix}%{version}-%{release}
+Recommends:	ceph-mgr = %{_epoch_prefix}%{version}-%{release}
```

Comment:
This can't be right... ceph-mgr should recommend ceph-mgr?

Reply:
sorry for that!